internal/proxy/proxy.go

```diff
@@ -289,6 +311,15 @@ func (c *Client) serveSocketMount(ctx context.Context, s *socketMount) error {
 		go func() {
 			c.cmd.Printf("[%s] accepted connection from %s\n", s.inst, cConn.RemoteAddr())
 
+			count := atomic.AddUint32(&c.connCount, 1)
```

Comment:
Is 32 the right choice? Looks like we are using 64 in v1 at the momment

Reply:
I guess I'll just match v1. uint64 seems to allow an impossibly high amount of connections, but I probably should toe the line of matching v1 behavior.